app/assets/javascripts/hyrax/dashboard_actions.js

```diff
@@ -32,4 +32,27 @@ Blacklight.onLoad(function() {
     return false;
   });
 
+  // Create sortable, searchable table
+  $('#analytics-collections-table').DataTable();
```

Comment:
This code, is not about the dashboard page itself, but about an analytics component that is on the dashbord page. Can you move this code to it's own file that is only about the analytics component of the page?